res/matrix-org/invite_template.eml

```diff
@@ -1,25 +1,25 @@
-Date: %(date)s
```

Comment:
How about we rename these Jinja2 templates to `.eml.j2` so it's easy to distinguish them?
(For the time being, I thought the plan was to keep the old system around for backwards compatibility)

Reply:
.j2 extension also confers the benefits of syntax highlighting in text editors (without having to fiddle with the settings)